.travis.yml

```diff
@@ -0,0 +1,69 @@
+language: cpp
+
+script:
+ - "bash -ex .travis-ci.sh"
+
+matrix:
+  fast_finish: true
+  include:
+    - compiler: clang
```

Comment:
I was going to leave it actually, because between clang and gcc we get a better selection of testing; we sometimes get failures on one and not the other and users may be running either.